dispatch/dispatch.go

```diff
@@ -111,6 +112,7 @@ func (d *Dispatcher) Groups() AlertOverview {
 			alertGroup, ok := seen[ag.fingerprint()]
 			if !ok {
 				alertGroup = &AlertGroup{Labels: ag.labels}
+				alertGroup.GroupKey = uint64(ag.labels.Fingerprint())
```

Comment:
It's actually generated differently: https://github.com/prometheus/alertmanager/blob/master/dispatch/dispatch.go#L332.

Probably best to at a `GroupKey() uint64` method to the aggregation group to avoid getting out of sync.